### ISSUE TITLE
Pump version in footer

### ISSUE THREAD

**Issue by vxcamiloxv** [pump-io/pump.io#1436]
For user reference (for reporting bugs) I think is good idea display version on the footer similar to **mastodon** or **diaspora**, for developer is easy inspect the request headers, but for the user is hard

**Comment by strugee** [comment]
Yeah, I can see a use for this. Probably just change the bit about the license to include a version number.

It might be useful to know if the admin was running a version with local patches though... that might be tricky to do.

**Comment by vxcamiloxv** [comment]
Check patches could be great, really is not tricky to do but could be slow to startup when comparte the folder checksum with the original code.

libraries: 
- https://www.npmjs.com/package/folder-hash
- https://github.com/mcavage/node-dirsum

**Comment by strugee** [comment]
Actually I'm changing my mind on that and thinking that it introduces too much complexity... instead I think we should write a guide that details how to patch the source and what things need to be adjusted. It'd mention adjusting this to reflect any changes made.

**Comment by strugee** [comment]
Filed https://gitlab.com/pump.io/pump-docs/issues/10/

**Comment by vxcamiloxv** [comment]
Yes, is much complexity and slow, and anyone can change the code and disable the validation

**Comment by strugee** [comment]
#1501 suggests adding this to the settings instead of the footer. I think that might look cleaner; what do you think @vxcamiloxv?

**Comment by vxcamiloxv** [comment]
The version in the footer is only visible in sections without infinite scroll like "account", "settings" or main page, the version in the footer can help to the user know if the instance when he/she will open account has a recently version.

in web platforms like github the footer has useful links like "privacy policy" "api" in major free/open source software projects the footer has the link to the "source code".

**Comment by Serkan-devel** [comment]
Maybe the commit hash from the repo could also be displayed next to the version

**Comment by strugee** [comment]
Why? That only seems useful for alphas, if at all

**Comment by Serkan-devel** [comment]
https://derpibooru.org/ they have it like that and it looks reasonable

**Comment by strugee** [comment]
Is there a compelling reason for pump.io to do this, though?

**Comment by Serkan-devel** [comment]
Maybe it could show if it has been patched from upstream or has its own patches

**Comment by Serkan-devel** [comment]
fair enough

**Comment by vxcamiloxv** [comment]
btw, the settings page is more a "profile" page, that page should rename because don't have any configuration option to be a "settings" page, all fields there change the user/profile object
